nextjs-13/src/lib/request/getError.ts

```diff
@@ -0,0 +1,11 @@
+import { StatusCodes } from 'http-status-codes';
+import { NextResponse } from 'next/server';
+
+function getError(error) {
```

Comment:
It sounds odd to name this `get` since it is not respond with a JSON response. How about:

```suggestion
function errorResponse(error) {
```

In addition, it might be clear to name it more specifically after the default error code:

```suggestion
function badRequestResponse(error) {
```

Also, in terms of code structure, while having single-method in each file is great for code discovery, it could quickly get out of hand on a large project. Also it makes it harder to see which code is related together. What do you think of group all error responses method in the same file:

```
├── lib/request
│   └── error.ts
```

Then in `error.ts`:

```typescript

import { StatusCodes } from 'http-status-codes';
import { NextResponse } from 'next/server';

export const errorMessageList = {
  [StatusCodes.UNPROCESSABLE_ENTITY]: 'Invalid params'
}

export function badRequestResponse(error) {
  return NextResponse.json(
    { message: error },
    { status: StatusCodes.UNPROCESSABLE_ENTITY }
  );
}

export function invalidParamsResponseError() {
  return badRequestResponse(errorMessageList[StatusCodes.UNPROCESSABLE_ENTITY]);
}
```

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/121/commits/6b4ce7ed40366c3489eabff5d924bc05688f514c#diff-83ff6fcba19133c2c5bf50957cd28828711e480cc9a940e83370a1a2c50d9c44R1-R21)